src/reducers/index.js

```diff
@@ -1,7 +1,7 @@
 import { combineReducers } from 'redux';
-import playerReducer from './playerReducer';
+import player from './playerReducer';
```

Comment:
Dou a mesma sugestão citada acima, de separar as importações!!
```suggestion

import player from './playerReducer';
```